wagmi.config.ts

```diff
@@ -36,7 +36,17 @@ function sleep(ms: number = 1_000) {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
+// @ts-ignore
+mainnet.rpcUrls.default.http[0] = 'https://mainnet.gateway.tenderly.co';
+// @ts-ignore
+mainnet.rpcUrls.public.http[0] = 'https://mainnet.gateway.tenderly.co';
+// @ts-ignore
+sepolia.rpcUrls.default.http[0] = 'https://sepolia.gateway.tenderly.co';
+// @ts-ignore
+sepolia.rpcUrls.public.http[0] = 'https://sepolia.gateway.tenderly.co';
```

Comment:
This is because both `https://cloudflare-eth.com` and `https://rpc.sepolia.org` have issues.
I know it's a bit dirty, but we very rarely run this (only when Nitro contracts or ArbOS is bumped), and this is a quick fix.